go.mod

```diff
@@ -1,6 +1,6 @@
 module gorm.io/gorm
 
-go 1.14
+go 1.16
```

Comment:
Tentatively updated according to min version on CI.
https://github.com/zaneli/gorm/blob/4768b87f15cdfff699fb449cedb276d185ce8763/.github/workflows/tests.yml#L19